### DISCUSSION TITLE
我出现了最可怕的问题

### DISCUSSION THREAD

**Discussion by wsq1211278323** [antoor/antSword#339] [Q&A]
我初始化后打开只显示一只猫啊！！！用不了。。。

**Comment by pfinal-nc** [comment]
什么系统 什么环境 具体描述一下

**Reply by wsq1211278323** [reply]
AntSword-Loader-v4.0.3-win32-x64
我电脑的系统是win11。2023 年 7 月 11 日 -KB5028851 .NET Framework 3.5 和 4.8.1 的累积更新（适用于 Windows 11 版本 22H2）
就是压缩包解压后，我就打开文件，选择蚂蚁剑的exe文件，然后进行初始化，选择了那个代码后就显示初始化成功了。当我重启的时候，蚂蚁剑就一直显示一个图标，一只猫拿着一把剑的图标。就一直不动了。

**Reply by wsq1211278323** [reply]
目录也纯英目录，但就是打不开

**Comment by wsq1211278323** [comment]
田冶
***@***.***



&nbsp;


是这个版本的电脑系统，蚂蚁剑是老师发的------------------&nbsp;原始邮件&nbsp;------------------




发件人:                                                                                                                        "AntSwordProject/antSword"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年7月31日(星期一) 上午9:14
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [AntSwordProject/antSword] 我出现了最可怕的问题 (Discussion #339)





 
什么系统 什么环境 具体描述一下

**Comment by pfinal-nc** [comment]
已尝试 在 win11 上打开 是正常的, 不确定 你这边是什么情况

**Comment by wsq1211278323** [comment]
天要亡我😭发自我的荣耀手机-------- 原始邮件 --------发件人： PFinal南丞 ***@***.***>日期： 2023年7月31日周一 14:36收件人： AntSwordProject/antSword ***@***.***>抄送： wsq1211278323 ***@***.***>, Author ***@***.***>主    题： Re: [AntSwordProject/antSword] 我出现了最可怕的问题 (Discussion #339)
已尝试 在 win11 上打开 是正常的, 不确定 你这边是什么情况
